web/person-roles/app/components/Form/OfficerChange/index.vue

```diff
@@ -74,7 +74,7 @@ const schema = z.object({
   lastName: z.string().min(1, t('validation.fieldRequired')).max(30, t('validation.maxChars', { count: 30 })),
   preferredName: z.string().max(50, t('validation.maxChars', { count: 50 })).default(''),
   hasPreferredName: z.boolean(),
-  mailingAddress: getNotRequiredAddressSchema(),
+  mailingAddress: getRequiredAddressSchema(),
```

Comment:
Same schema used by deliveryAddress (below).

If you click "same as", the mailing address component is not rendered and not validated.